plugins/source/scheduler.go

```diff
@@ -144,11 +145,19 @@ func (p *Plugin) periodicMetricLogger(ctx context.Context, wg *sync.WaitGroup) {
 		case <-ctx.Done():
 			return
 		case <-ticker.C:
-			p.logger.Info().
+			inProgressTables := p.metrics.InProgressTables()
+
+			logLine := p.logger.Info().
 				Uint64("total_resources", p.metrics.TotalResourcesAtomic()).
 				Uint64("total_errors", p.metrics.TotalErrorsAtomic()).
 				Uint64("total_panics", p.metrics.TotalPanicsAtomic()).
-				Msg("Sync in progress")
+				Int("num_in_progress_tables", len(inProgressTables))
+
+			if len(inProgressTables) <= periodicMetricLoggerLogTablesLimit {
+				logLine.Strs("in_progress_tables", inProgressTables)
+			}
```

Comment:
I think we should have an `else` case here, so that these logs still appear if there are many tables getting synced (as that can be a clue as well). Maybe something like 

```
logLine.Strs("in_progress_tables", inProgressTables[:periodicMetricLoggerLogTablesLimit])
logLine.Bool("truncated", true)
```

(And we can put `logLine.Bool("truncated", false)` in the other case). WDYT?